src/components/Games/story-game/completion-screen.jsx

```
@@ -3,6 +3,7 @@ import { AiFillTrophy } from "react-icons/ai";
 import { FaSyncAlt } from "react-icons/fa";
 
 const CompletionScreen = ({ score, total, onRestart }) => {
+  const percentage = Math.round((score / total) * 100);
```

ESLint

[error] 'percentage' is assigned a value but never used.

(no-unused-vars)